modules/apps/transfer/ibc_module.go

```diff
@@ -170,6 +172,27 @@ func (IBCModule) OnChanCloseConfirm(
 	return nil
 }
 
+func (IBCModule) getMultiDenomFungibleTokenPacketData(bz []byte) (multidenom.FungibleTokenPacketData, error) {
```

Comment:
I think it might be beneficial to reference the specific version of ics20 (or atleast refer to multidenom) in the function name as a conversion in the logic is happening and it's not simply a getter. maybe using `get` in the name is already too misleading? maybe something more like `unmarshalPacketDataBytesToICS20V2`, or something to that effect

Reply:
yeah, I think I like `unmarshalPacketDataBytesToICS20V2` or something along those lines!

Reply:
I hope you don't mind: I liked the suggestion and renamed the function. I am sure we will still change the name a couple of times, but for now sounds good to me. :)